kamperiadis/azure-cli
# add vnet warning logics when provide managed environment

**Related command**
Add a warning message and ignore the VNET and SUBNET parameters, when managed environement and VNET or SUBNET parameter are both provided when create a function app using `az functionapp create` command. 

**Description**


For project Centauri, when customer provide the managed environment while creating function app, they are supposed to not provide any VNET and SUBNET parameters. 
If above happen, we add a warning message and ignore the VNET and SUBNET parameters. 

**Testing Guide**

To be updated. 


---

This checklist is used to make sure that common guidelines for a pull request are followed.
